public/index.php

```diff
@@ -21,15 +21,16 @@
     $_SERVER['SCRIPT_NAME'] = '/index.php';
 }
 
-require_once dirname(__DIR__) . '/preload.php';
+chdir(dirname(__DIR__));
+require_once 'preload.php';
 
 if (getenv('YII_ENV') === 'test') {
-    $c3 = dirname(__DIR__) . '/c3.php';
+    $c3 = 'c3.php';
     if (file_exists($c3)) {
         require_once $c3;
     }
 }
 
 // Run HTTP application runner
-$runner = new HttpApplicationRunner(dirname(__DIR__), $_ENV['YII_DEBUG'], $_ENV['YII_ENV']);
+$runner = new HttpApplicationRunner(getcwd(), $_ENV['YII_DEBUG'], $_ENV['YII_ENV']);
```

Comment:
What is the reason for this change?

Reply:
try to compare current working dir in cases:
- `php yii serve`
- `php -S 127.0.0.1:8080 -t public` (like in [tests](https://github.com/yiisoft/demo/blob/15504bd4383685f2a095e857448d5c274315fab0/.github/workflows/build.yml#L65))

Reply:
That won't work if you'll launch `./yii` not from the same directory as it is in:

```sh
./apps/yii3_demo/yii serve
```

Reply:
> That won't work

Feel free to fix it